services/brig/src/Brig/API/Error.hs

```diff
@@ -425,13 +425,12 @@ noBindingTeam = Wai.Error status403 "no-binding-team" "Operation allowed only on
 sameBindingTeamUsers :: Wai.Error
 sameBindingTeamUsers = Wai.Error status403 "same-binding-team-users" "Operation not allowed to binding team users."
 
-noOtherOwner :: Wai.Error
-noOtherOwner =
+deleteeIsOwner :: Wai.Error
+deleteeIsOwner =
```

Comment:
This error is supposed to mean that the deletee and the deleter are both same person and owner, I think we should name it better, something like `ownerDeletingSelf`

Reply:
I say it's actually about the owner role.  Let's discuss this.